hosting_docs/source/index.rst

```diff
@@ -25,4 +25,5 @@ This documentation contains guides and reference material on how to self host Co
    Service Guides <services/0-index>
    Backups and Disaster Recovery <backups_dr/1-index>
    Security <security/1-security>
+   Support System <support/1-index>
```

Comment:
We can call this and [this](https://github.com/dimagi/commcare-cloud/pull/5016/files#diff-b8052816d23e4171b69ed21d90994a27cb0481a7927942df3e58786ac46051c3R1) 'Supporting Your Users'.